SingularityBase/src/main/java/com/hubspot/singularity/executor/SingularityExecutorLogrotateFrequency.java

```diff
@@ -0,0 +1,19 @@
+package com.hubspot.singularity.executor;
+
+public enum SingularityExecutorLogrotateFrequency {
+    HOURLY("monthly"),
```

Comment:
meant to add a comment here -- `hourly` isn't supported by all versions of logrotate. my intent was to make it run as infrequently as possible (hence `monthly`) and then manually trigger logrotate by the executor. (open to other options if you have other ideas here)